modules/codegen/src/smithy4s/codegen/internals/Renderer.scala

```diff
@@ -792,6 +808,7 @@ private[internals] class Renderer(compilationUnit: CompilationUnit) { self =>
       hints: List[Hint]
   ): Lines = lines(
     deprecationAnnotation(hints),
+    documentationAnnotation(hints),
```

Comment:
Looks like I accidentally made it easier for you by putting all these `deprecationAnnotation` calls in - happy if it saved you some time :)

Reply:
It served as an awesome example, and I was like "is it really this easy to add the annotation"? It also helped clarify the structure of these `renderToX` functions.

Reply:
I'm also realizing that `documentationAnnotation` should be _above_ `deprecationAnnotation`, because you'll want it to be like:
```scala
/** Now deprecated, used to do something useful */
@deprecated
def destroyEverything()
```

Reply:
good point, yeah